docs/en/resources/tools/dataform/dataform-init-local.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "dataform-init-local"
+type: docs
+weight: 1
+description: > 
+  A "dataform-init-local" tool runs the `dataform init` CLI command on a local project directory.
+aliases:
+- /resources/tools/dataform-init-local
+---
+
+## About
+
+A `dataform-init-local` tool runs the `dataform init` command on a directory in the local file system.
+
+It is a standalone tool and **is not** compatible with any sources.
+
+At invocation time, the tool executes `dataform init` in the specified project directory to set up the corresponding files for a Dataform project.
+
+`dataform-init-local` takes the following parameter: 
+- `project_dir` (string): The absolute or relative path to the local Dataform project directory. The server process must have write access to this path.
+- `default_database` (string): The default database to use, equivalent to Google Cloud Project ID.
```

Comment:
Do these inputs need to have the "default" prefix? I'm not sure about dataform's usage but these seem verbose.